subsys/mgmt/ec_host_cmd/backends/ec_host_cmd_backend_shi_npcx.c

```diff
@@ -346,11 +346,6 @@ static void shi_npcx_bad_received_data(const struct device *dev)
 	/* SHI receive bad data */
 	LOG_WRN("SHIBD");
 	LOG_HEXDUMP_DBG(data->in_msg, data->rx_ctx->len, "in_msg=");
-
```

Comment:
Can you add a comment that the state machine is reset after CS de-assertion?

Reply:
Done.